torch/distributed/rpc.py

```diff
@@ -105,7 +113,7 @@ def rpc(to, func, args=None, kwargs=None, async_call=False):
 
     Arguments:
         to (str): name of the destination worker.
-        func (callable): a builtin function (e.g., ``torch.add``).
+        func (callable): a builtin function or user defined python function (e.g., ``torch.add``).
```

Comment:
Another way to spell the docs here, is to just say any callable is fine, but then say that builtin functions (like `torch.add`) can be sent over RPC more efficiently.